cqc/pythonLib.py

```diff
@@ -218,6 +467,9 @@ def __init__(self, name, socket_address=None, appID=None, pend_messages=False,
             :param network_name: None or str
                 Used if simulaqron is used to load socket addresses for the backend
         """
+
+        super().__init__(pend_messages=pend_messages)
+
```

Comment:
The setting of `self._opened_with_with` below should probably move to the abstract class since it it checked when creating a qubit.

Reply:
Done 89842ae348ed36d9bb04a5ffb7fbfd9b74195794